tests/ui/features/@vulnerability-explorer/vulnerability-explorer.step.ts

```diff
@@ -16,6 +17,55 @@ Given(
   }
 );
 
+// Vulnerability Explorer
+Then(
+  "The severity is {string} and the CVSS score is {string}",
+  async ({ page }, severityDescription, severityScore) => {
+    if (severityDescription == "Unknown") {
+      var severity = `${severityDescription}`;
+    } else {
+      var severity = `${severityDescription}(${severityScore})`;
+    }
+    await expect(page.getByText(severity).first()).toBeVisible();
+  }
+);
+
+Then(
+  "The description begins with {string}",
+  async ({ page }, descriptionBeginsWith) => {
+    await expect(
+      page.getByRole("paragraph").filter({ hasText: descriptionBeginsWith })
+    ).toBeVisible();
+  }
+);
+
+Then(
+  "The Reserved date is {string}, the Published date is {string} and Last modified date is {string}",
+  async ({ page }, dateReserved, datePublished, dateLastModified) => {
+    await expect(
+      page
+        .locator(".pf-v5-c-description-list > div:nth-child(1)")
+        .filter({ hasText: "Reserved" })
+        .filter({ hasText: dateReserved })
+        .first()
+    ).toBeVisible();
+    await expect(
+      page
+        .locator(".pf-v5-c-description-list > div:nth-child(2)")
+        .filter({ hasText: "Published date" })
```

Comment:
```suggestion
        .filter({ hasText: "Published" })
```
text changed to just "Published"